tutorials/fundamentals/pop.md

```diff
@@ -0,0 +1,22 @@
+The `pop()` functions removes the last element from the array it is called upon and returns the popped element.
+
+```javascript
+const array = [1, 2, 3, 4, 5, 6];
+array.pop(); // 6;
+array; // 1, 2, 3, 4, 5
+```
+
+This function shortens the length of the array and returns `undefined` if the array is empty, similarly to [`shift()`](/tutorials/fundamentals/shift).
+
+```javascript
+const array = [1, 2, 3, 4, 5, 6];
+
+array.length; // 6
+array.pop(); // 6;
+array; // 1, 2, 3, 4, 5
+array.length; // 5
+
+const emptyArray = [];
+emptyArray.pop(); // undefined
+emptyArray.length; // 0
+```
```

Comment:
Add an example of using `push()` and `pop()` to use the array as a stack, like how `shift()` demonstrated how to use an array as a queue.

Who said you'd never use undergrad data structures in real life? :)